raft/src/main/java/org/apache/kafka/raft/ConnectionCache.java

```diff
@@ -160,6 +160,10 @@ boolean isReady(long timeMs) {
             return state == State.READY;
         }
 
+        State state() {
```

Comment:
Was this needed for something?

Reply:
It's only for debugging purpose. Could remove.